indra/llui/llslider.cpp

```diff
@@ -279,12 +281,31 @@ BOOL LLSlider::handleKeyHere(KEY key, MASK mask)
 
 BOOL LLSlider::handleScrollWheel(S32 x, S32 y, S32 clicks)
 {
-    if ( mOrientation == VERTICAL )
+    // <FS> [FIRE-30873]:  BD - UI Improvements: wheel works on both axes with ctrl, ctrl+shift+wheel moves at 10% sensitivity
+    SHORT shiftKeyState = GetAsyncKeyState(VK_SHIFT);
+    SHORT ctrlKeyState = GetAsyncKeyState(VK_CONTROL);
```

Comment:
These are hacky.
I looked at injecting MASK, and tried, but boy, I could have gone all the way back to base, allowing mask to be offered everywhere... which isn't a bad thing.

What I worry about is if this is platform dependent. I think Mac users won't like CTRL.